scalastyle-config.xml

```diff
@@ -1,6 +1,6 @@
 <scalastyle commentFilter="enabled">
```

Comment:
Slightly different config but happy for it to be left

[scalastyle-config.xml.zip](https://github.com/hmrc/ras-frontend/files/12302455/scalastyle-config.xml.zip)